test/uk/gov/hmrc/tai/service/IncomeServiceSpec.scala

```diff
@@ -98,12 +98,46 @@ class IncomeServiceSpec extends BaseSpec {
           BigDecimal(0))
       )
 
+      val employment = Employment(
+        "company name",
+        Live,
+        Some("888"),
+        new LocalDate(TaxYear().next.year, 5, 26),
+        None,
+        Nil,
+        "",
+        "",
+        1,
+        Some(100),
+        hasPayrolledBenefit = false,
+        receivingOccupationalPension = true
+      )
+      val employment2 = Employment(
+        "company name",
+        Live,
+        Some("888"),
+        new LocalDate(TaxYear().next.year, 5, 26),
+        None,
+        Nil,
+        "",
+        "",
+        2,
+        Some(100),
+        hasPayrolledBenefit = false,
+        receivingOccupationalPension = true
+      )
+
+      val mockEmploymentService = mock[EmploymentService]
       val mockIncomeRepository = mock[IncomeRepository]
       when(mockIncomeRepository.taxCodeIncomes(any(), any())(any()))
         .thenReturn(Future.successful(taxCodeIncomes))
 
-      val SUT = createSUT(incomeRepository = mockIncomeRepository)
-      val result = SUT.taxCodeIncomes(nino, TaxYear())(HeaderCarrier()).futureValue
+      when(mockEmploymentService.employments(any[Nino], any[TaxYear])(any[HeaderCarrier]))
+        .thenReturn(Future.successful(Seq(employment, employment2)))
+
+      val sut = createSUT(employmentService = mockEmploymentService, incomeRepository = mockIncomeRepository)
+      val result = sut.taxCodeIncomes(nino, TaxYear())(HeaderCarrier()).futureValue
+
 
       result mustBe taxCodeIncomes
     }
```

Comment:
is it worth writing a test to check the employmentStatus that we are getting back or has this been done already?

Reply:
i think it's worth creating a test that returns an empty list when calling: `employmentService.employments(nino, year)`
and then checking it still has a status which is the default one that comes from `incomeRepository.taxCodeIncomes(nino, year)`

Reply:
Sounds good to me, what about an AT? e.g. check that if we have a ceased status then the user sees a ceased employment?

Reply:
this has been done in the first test. The employment API returns CEASED therefore overriding the Tax API

Reply:
agreed